app/_redirects

```diff
@@ -839,4 +839,4 @@
 #3.2
 
 /gateway/latest/admin-api/index/#keys-entity     /gateway/latest/admin-api/keys-object
-/gateway/latest/admin-api/index/#vaults-entity  /gateway/latest/admin-api/vaults-object
\ No newline at end of file
+/gateway/latest/admin-api/index/#vaults-entity  /gateway/latest/admin-api/vaults-object
```

Comment:
Looks like this one still points to "/gateway/latest/admin-api/index/#vaults-entity", but the page isn't broken, just takes you to the top of the Admin API page.